diff --git a/.gitignore b/.gitignore
index 9fa57fb..582c388 100644
--- a/.gitignore
+++ b/.gitignore
@@ -23,6 +23,10 @@ hw/core-v-mini-mcu/include/core_v_mini_mcu_pkg.sv
 hw/core-v-mini-mcu/system_bus.sv
 hw/core-v-mini-mcu/system_xbar.sv
 hw/core-v-mini-mcu/memory_subsystem.sv
+hw/core-v-mini-mcu/ao_peripheral_subsystem.sv
+hw/core-v-mini-mcu/core_v_mini_mcu.sv
+hw/core-v-mini-mcu/peripheral_subsystem.sv
+hw/core-v-mini-mcu/cpu_subsystem.sv
 
 hw/system/x_heep_system.sv
 hw/system/pad_ring.sv
